libsolidity/Types.h

```diff
@@ -369,6 +368,30 @@ class StructType: public Type
 };
 
 /**
+ * The type of an enum instance, there is one distinct type per enum definition.
+ */
+class EnumType: public Type
+{
+public:
+	virtual Category getCategory() const override { return Category::Enum; }
+	explicit EnumType(EnumDefinition const& _enum): m_enum(_enum) {}
+	virtual TypePointer unaryOperatorResult(Token::Value _operator) const override;
+	virtual bool operator==(Type const& _other) const override;
+	virtual unsigned getSizeOnStack() const override { return 1; /*@todo*/ }
```

Comment:
`1` is always correct. We *could* explicitly force the number of members to be less than 2^256, though...